inference-engine/src/vpu/graph_transformer/include/vpu/middleend/allocator/structs.hpp

```diff
@@ -21,6 +21,7 @@ namespace vpu {
 const int DDR_MAX_SIZE = 512 * 1024 * 1024;
 const int CMX_SLICE_SIZE = 128 * 1024;
 const int DATA_ALIGNMENT = 64;
+const int CMX_BUFFER_SIZE = 100 * 1024;
```

Comment:
Why buffer of that size? Is there some meaning under 100 and 1024 ?

Reply:
Renamed this variable to clarify its meaning. It is the size of CMX memory each SHAVE can use.